install_config/registry/extended_registry_configuration.adoc

```diff
@@ -581,6 +586,35 @@ be safe to enable its support in the registry. See the
 xref:docker-registry-configuration-reference-middleware[middleware
 configuration reference] above for particular option.
 
+[[docker-registry-configuration-reference-openshift]]
+=== OpenShift
+
+This section allows to configure global settings related to {product-title}
+specific features. It will obsolete all the `*openshift*` related settings in
+label:docker-registry-configuration-reference-middleware[middleware section] in
```

Comment:
Ok, I must have been doing something wrong before. I've rewritten it to `xref:`.